.github/workflows/scheduled.yml

```diff
@@ -60,8 +61,11 @@ jobs:
         id: set-up-homebrew
         uses: Homebrew/actions/setup-homebrew@master
 
+      - name: Reset homebrew/core tap
+        run: brew update-reset $(brew --repository homebrew/core)
```

Comment:
Are you sure this is necessary? It's really slow.

Reply:
Maybe not, do you think it's necessary for the `homebrew/cask` tap?

Reply:
@Rylan12 Yeh, it's needed there but not here I think because `setup-homebrew` sets up homebrew/core and not homebrew/cask.